decisions/World Bank.txt

```diff
@@ -23,13 +23,6 @@ political_decisions = {
 		allow = {}
 		
 		effect = {
-			# For the World Bank
-			any_owned = {
-				add_province_modifier = {
-					name = bank_cheats
-					duration = -1
-				}
-			}
```

Comment:
Comment this out instead -- it might be something we want later on to help with economy balancing

Reply:
Tried to do it from phone and screwed up the encoding by choosing CRLF linebreak because it's supposedly for Windows